api/v1alpha1/cluster_defaults_test.go

```diff
@@ -18,19 +18,21 @@ package v1alpha1
 
 import (
 	"fmt"
+	"testing"
+
 	"github.com/google/go-cmp/cmp"
 	"github.com/stretchr/testify/assert"
-	"testing"
 )
 
 func TestSetClusterSpecDefaults(t *testing.T) {
 	s := &CrdbClusterSpec{}
-
+	max := int32(1)
```

Comment:
nit: probably this variable could be better named as it is unclear what max is it of.

Reply:
Updated